src/khoj/routers/web_client.py

```diff
@@ -166,6 +168,7 @@ def config_page(request: Request):
             "current_model_state": successfully_configured,
             "anonymous_mode": state.anonymous_mode,
             "username": user.username,
```

Comment:
Isn't the username currently always set to the user's email address? If so, should we just use that for the user name (instead of adding the new given_name field) or populate the `username` field with the username from gmail to make the field more useful?

Reply:
The usage of `username` is semantically more for unique identification, rather than common name. I'll leave as-is.